src/store/iter.rs

```diff
@@ -186,21 +186,15 @@ mod tests {
             parent: &ErrorStore,
             bounds: (Bound::Unbounded, Bound::Unbounded),
             done: false,
-        }; 
-        assert_eq!(
-            iter.next().unwrap().unwrap_err().to_string(),
-            "get",
-        );
+        };
+        assert_eq!(iter.next().unwrap().unwrap_err().to_string(), "get",);
```

Comment:
Weird that rustfmt didn't take out the trailing comma in this and the next assert, but let's get rid of that.

Reply:
Again, this is a bit concerning. I think I'll open two separate issues in the fmt repo tomorrow for the weird cases that we ran into.